app/models/postcard.rb

```diff
@@ -5,6 +5,8 @@ class Postcard < ApplicationRecord
   enum status: %i[in_transit in_local_area processed_for_delivery re_routed
                   returned_to_sender]
 
+  PRICE = 300
```

Comment:
I don't like this, but I think it's OK for V1.

Ideally, we'd be able to change pricing and have a source of truth for when we changed it. Maybe have the ability to grandfather some users in to an old pricing scheme. 

It would also be ideal to be able to change pricing without a code change. We could accomplish this by making storing price in ENV. This feels hacky as well, but could be the lesser of 2 evils? 🤷‍♂ 

@edance what do you think?

Reply:
I like the `ENV` variable idea